storybook/pages/SharedAddressesPopupPage.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml 2.15
+
+import Storybook 1.0
+import Models 1.0
+
+import AppLayouts.Communities.popups 1.0
+
+SplitView {
+    id: root
+    Logs { id: logs }
+
+    ListModel {
+        id: emptyModel
+    }
+
+    Component {
+        id: dlgComponent
+        SharedAddressesPopup {
```

Comment:
Some warning is printed here:

https://github.com/status-im/status-desktop/assets/20650004/6e0440c5-a21a-407e-a4a6-f9b6fb51b17a